### DISCUSSION TITLE
q-select's label binding doesnt work

### DISCUSSION THREAD

**Discussion by sjoshid** [quasarframework/quasar#16605] [General - Components / Directives / etc]
I have a simple select with following code:
`<q-select v-model="model" :options="options" dense filled :label="aValidJSVar"/>`

But I dont see label floating on top of select. 

But this works
`<q-select v-model="model" :options="options" dense filled label="stringLiteral"/>`

Why?

**Comment by pdanpdan** [comment]
your `aValidJSVar` is empty

**Reply by sjoshid** [reply]
It's not. I guarantee you.

**Reply by sjoshid** [reply]
BTW `aValidJSVar` is a mandatory prop. I really hope that doesn't change anything.

**Comment by pdanpdan** [comment]
Mandatory doesn't mean:
- it is not an empty string
- you didn't mistyped it
- ...
